sdk/communication/communication-jobrouter/test/public/exceptionPolicies.spec.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder } from "@azure-tools/test-recorder";
+// import { assert } from "chai";
+import { createRecordedRouterClientWithConnectionString } from "../internal/utils/mockClient";
+import { Context } from "mocha";
+// import { exceptionPolicyRequest } from "./utils/testData";
+// import { RouterAdministrationClient } from "../../src";
+
+describe("RouterClient", function() {
+  let recorder: Recorder;
+  // let client: RouterClient;
+  // let administrationClient: RouterAdministrationClient;
+
+  describe("Exception Policy Operations", function() {
+    beforeEach(async function(this: Context) {
+      ({ recorder } = await createRecordedRouterClientWithConnectionString(this));
+    });
+
+    afterEach(async function(this: Context) {
+      if (!this.currentTest?.isPending() && recorder) {
+        await recorder.stop();
+      }
+    });
+
+    // exception policy actions
```

Comment:
Why are all these commented out?

Reply:
Crate an issue for it, https://github.com/Azure/azure-sdk-for-js/issues/23007